server/src/main/java/org/apache/druid/client/cache/HybridCache.java

```diff
@@ -131,6 +134,15 @@ public void close(String namespace)
     level2.close(namespace);
   }
 
+  @Override
+  @LifecycleStop
+  public void close() throws IOException
+  {
+    try (Closeable closeable1 = level1; Closeable closeable2 = level2) {
```

Comment:
Yet could you extract this as a method? There are 5 more similar places in production code, you could find them using "structural search" with the following pattern:
```
$a$.close();
$b$.close();
```

And you could also add a corresponding "Structural Search Inspection" to prohibit adding more in the future.

Reply:
OK, I'll do that, thanks.

Reply:
ok

Reply:
Please create a PR or an issue

Reply:
Have created an issue.